packages/turbo-types/src/types/config.ts

```diff
@@ -80,11 +80,11 @@ export interface Pipeline {
    * Note: turbo automatically logs stderr/stdout to .turbo/run-<task>.log. This file is
    * always treated as a cacheable artifact and never needs to be specified.
    *
-   * Passing an empty array can be used to tell turbo that a task is a side-effect and
+   * Omitting this key or passing an empty array can be used to tell turbo that a task is a side-effect and
```

Comment:
We provide no control over caching of logs. If you cache the task, we cache the logs. We cache by default. Therefore, unless you specify `cache: false` or `--no-cache` this whole discussion is weird. Given that there is no longer any reason to pass `[]` here (prevent capturing the things in the default) we should probably just drop this discussion.

What about:
> Turborepo caches the logs of all tasks that are cached. Additionally, if you wish to capture file-system artifacts, you may specify